packages/SwingSet/test/metering/metered-code.js

```diff
@@ -3,7 +3,9 @@ function sub() {
 }
 
 function delveForeverIntoTheRecursiveDepths() {
-  return delveForeverIntoTheRecursiveDepths();
+  return (
+    delveForeverIntoTheRecursiveDepths() + delveForeverIntoTheRecursiveDepths()
```

Comment:
Would be neat to leave a comment tail-recursion optimization, specifically that XS actually does it!